modules/admin_manual/pages/configuration/server/occ_commands/app_commands/_ldap_integration_commands.adoc

```diff
@@ -37,146 +60,252 @@ Broaden the search to find, for example, `jeroboam` with the asterisk wildcard:
+=== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `configID`
+| Create a configuration with the specified id
+|===
+
+Configurations that you create without assigning a <configID> are automatically assigned IDs.
 
 [source,console,subs="attributes+"]
 ----
 {occ-command-example-prefix} ldap:create-empty-config
    Created new configuration with configID 's01'
 ----
 
-Then you can list and view your configurations:
+== List and View Your Configurations
+
+You can list and view your configurations:
 
 [source,console,subs="attributes+"]
 ----
-{occ-command-example-prefix} ldap:show-config
+{occ-command-example-prefix} ldap:show-config [options] [--] [<configID>]
 ----
 
-And view the configuration for a single `configID`:
+=== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `configID`
+| Will show the configuration of the specified id
+|===
+
+=== Options
+
+[width="100%",cols="20%,70%",]
+|===
+| `--show-password`
+| Show LDAP bind password
+
+| `--output[=OUTPUT]`
+| The output format to use (plain, json or json_pretty). [default: "plain"]
+|===
+
+View the configuration for a single `configID`:
 
 [source,console,subs="attributes+"]
 ----
 {occ-command-example-prefix} ldap:show-config s01
 ----
 
-`ldap:delete-config [configID]` deletes an existing LDAP configuration.
+== Delete an Existing LDAP Configuration
+
+Deletes an existing LDAP configuration.
+
+[source,console,subs="attributes+"]
+----
+ ldap:delete-config <configID>
+----
+
+=== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `configID`
+|  The configuration ID
+|===
 
 [source,console,subs="attributes+"]
 ----
 {occ-command-example-prefix} ldap:delete  s01
 Deleted configuration with configID 's01'
 ----
 
-The `ldap:set-config` command is for manipulating configurations, like this example that sets search attributes:
+== Invalidate LDAP Cache
+
+This command invalidates the LDAP cache for all users:
 
 [source,console,subs="attributes+"]
 ----
-{occ-command-example-prefix} ldap:set-config s01 ldapAttributesForUserSearch
-"cn;givenname;sn;displayname;mail"
+{occ-command-example-prefix} ldap:invalidate-cache
 ----
 
-The command takes the following format:
+== Manipulate LDAP Configurations
 
+This command manipulates LDAP configurations.
+
+[source,console,subs="attributes+"]
 ----
-ldap:set-config <configID> <configKey> <configValue>
+{occ-command-example-prefix} ldap:set-config <configID> <configKey> <configValue>
+----
+
+=== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `configID`
+|  The configuration ID
+
+| `configKey`
+|  The configuration key
+
+| `configValue`
+|  The new configuration value
+|===
+
+This example sets search attributes:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} ldap:set-config s01 ldapAttributesForUserSearch "cn;givenname;sn;displayname;mail"
 ----
 
 All of the available keys, along with default values for configValue, are listed in the table below.
```

Comment:
```suggestion
Available keys, along with default values for configValue, are listed in the table below.
```

Never say all. This table has 50 entries, My `occ ldap:show-config s01 | wc -l` reports 58 (including 4 non-body lines).